### DISCUSSION TITLE
Не работают некоторые приложения.

### DISCUSSION THREAD

**Discussion by umniqode** [remittor/zapret-openwrt#24] [Q&A]
Здравствуйте. Когда запущен Zapret - отваливаются некоторые программы. На андроид ТВ - билайн ТВ, на телефоне - ЕМИАС не работает. Может какие-то еще. Как только выключаю Zapret - все работает.
DNS-over-HTTPS настроил.
Подскажите, в какую сторону смотреть?

**Comment by MIzzzzon** [comment]
Нужно использовать конфигурацию с хостлистом исключений

**Reply by umniqode** [reply]
Я пока так сделал - Службы-Zapret-Настройки-MODE_FILTER изменил на autohostlist, по умолчанию стоял hostlist. Все работает.

**Comment by DmitryOlenin** [comment]
Может кто описать, как заставить работать MOS.RU или ЕМИАС?
Не помогло добавление в "User excluded hostname entries" 
emias.info
mos.ru
gosuslugi.ru
